smart_selects/static/smart-selects/admin/js/bindfields.js

```diff
@@ -24,6 +24,11 @@
         $.each($(".chained"), function (index, item) {
             initItem(item);
         });
+        $.each($(".filtered"), function (index, item) {
```

Comment:
There is a problem because when this is executed, the Horizontal way of the form is not ready, I made it work in this way:
`setTimeout(function() {
             $.each($(".filtered"), function (index, item) {
                if (item.hasAttribute('data-chainfield')) {
                    initItem(item);
                }
            });
        }, 1);`

Reply:
Just try with https://github.com/digi604/django-smart-selects/pull/273 - might help to trigger things in the right order